dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3JobTask.java

```diff
@@ -73,20 +75,21 @@ AtomicLong addDataTransferListener(final long totalJobSize) {
         return totalSent;
     }
 
-     void addWaitingForChunkListener(final long totalJobSize, final String targetDir) {
+    void addWaitingForChunkListener(final long totalJobSize, final String targetDir) {
         job.attachWaitingForChunksListener(retryAfterSeconds -> {
             for (int retryTimeRemaining = retryAfterSeconds; retryTimeRemaining >= 0; retryTimeRemaining--) {
                 try {
                     updateMessage(resourceBundle.getString("noAvailableChunks") + SPACE + retryTimeRemaining + resourceBundle.getString("seconds"));
                     Thread.sleep(1000);
                 } catch (final Exception e) {
-                    LOG.error("Exception in attachWaitingForChunksListener: {}", e);
+                    LOG.error("Failed : ", e);
```

Comment:
This error message should say what failed, ie, what were we trying to do when it failed.